wcivf/apps/elections/templates/elections/party_list_view.html

```diff
@@ -14,8 +14,12 @@
 
                 <h2>{{ party_name }}</h2>
                 <h4>
-                    {{ ballot.election.nice_election_name }}
-                    {% if not ballot.is_pcc and not ballot.is_mayoral %}: {{ ballot.friendly_name }}{% endif %}
+                    {% with ballot_url=ballot.get_absolute_url election_url=ballot.election.get_absolute_url %}
+                        <a href="{{ election_url }}"> {{ ballot.election.nice_election_name }} </a>
```

Comment:
really minor, but can we trim the whitespace here so the link "hugs" the text and we don't get underlined whitespace